parsil/src/tests.rs

```diff
@@ -149,6 +149,7 @@ fn test_serde_circuit_pis() {
 }
 
 #[test]
+#[ignore = "wait for non-aggregation SELECT to come back"]
```

Comment:
Why? this branch should be synced with simple SELECT queries branch, which should include non-aggregation SELECT.